### PR TITLE
cleanup sync errors related to directories

### PR DESCRIPTION
**Pull request by mpnowacki-reef**

### CODE REVIEW

**Comment by mpnowacki-reef** [comment]
waiting for https://github.com/reef-technologies/b2-sdk-python/pull/110

**Comment by mpnowacki-reef** [comment]
solves https://github.com/Backblaze/b2-sdk-python/issues/18

File: test/unit/v1/test_sync.py [hunk 1]
```diff
@@ -657,6 +657,77 @@ def _check_one(self, expected, to_parse):
         self.assertEqual(expected, str(parse_sync_folder(str(to_parse), api)))
 
 
+class TestFolderExceptions:
+    """There is an exact copy of this class in unit/v1/test_sync.py - TODO: leave only one when migrating tests to
```
Comment (mzukowski-reef) [inline test/unit/v1/test_sync.py:661]:
```suggestion
    """There is an exact copy of this class in unit/v0/test_sync.py - TODO: leave only one when migrating tests to
```
:P

**Review by mzukowski-reef — APPROVED** [review]

**Comment by mpnowacki-reef** [comment]
replaced by https://github.com/Backblaze/b2-sdk-python/pull/228

**Pull request closed without merging**